app/code/core/Mage/Adminhtml/Block/Customer/Edit/Tab/View.php

```diff
@@ -140,8 +140,8 @@ public function getStoreLastLoginDateTimezone()
     public function getCurrentStatus()
     {
         $log = $this->getCustomerLog();
-        if ($log->getLogoutAt() ||
-            strtotime(now())-strtotime($log->getLastVisitAt())>Mage_Log_Model_Visitor::getOnlineMinutesInterval()*60) {
+        if ($log->getLogoutAt() || (!empty($log->getLastVisitAt()) &&
```

Comment:
This fix removes the deprecation error but introduces a bug.  `strtotime(null)` evaluates to `false` so nothing is subtracted from `strtotime(now())` and the 2nd condition evaluates to true so 'offline' is returned. With your fix in those cases the method returns 'online' in those cases. 
Your solution e.g. displays: 
![image](https://user-images.githubusercontent.com/2728018/150175296-c064407c-1c83-4d50-8dad-879eb3c39216.png)
before it displayed:
![image](https://user-images.githubusercontent.com/2728018/150176127-6cfafeea-c98e-4663-ad63-b2f58f9bbae9.png)

I fixed it by combining your check with correctly executing the subtraction: 
```php
if ($log->getLogoutAt() 
    || strtotime(now()) - (!empty($log->getLastVisitAt()) ? strtotime($log->getLastVisitAt()) : 0) > Mage_Log_Model_Visitor::getOnlineMinutesInterval() * 60) {
    return Mage::helper('customer')->__('Offline');
}
```